### ISSUE TITLE
General Settings: required fields (asterisk)

### ISSUE THREAD

**Issue by troughc** [culturecreates/footlight-app#1202]
In General settings, even though image are marked as mandatory, I can save without setting them to anything.

![Image](https://github.com/culturecreates/footlight-app/assets/64489312/be795834-c815-46f9-9660-823189327d0d)

After I save, a 0 appears in maximum width. Nothing appears in the aspect ratio fields

These settings are mandatory so must be there. To make it easier for the user to save, please add default values in them that appear from the start.
Also, please note that **there are too many asterisks**, please do it this way instead (only mark the titles):

![Image](https://github.com/culturecreates/footlight-app/assets/64489312/64f35034-a17d-463b-ab70-7d33b41a6ae7)

**My recommended default settings, i.e. factory settings :) (to be displayed in the UI):**
Thumbnail
Aspect ratio: 3:2
Max width: 600

large image
Aspect ratio: 16:9
Max width 1200

If you have another opinion on aspect ratio, let me know.

**Comment by troughc** [comment]
@SyamBabu-M there are two issues, one with Artbi where I can save without adding a large image aspect ratio


![Image](https://github.com/culturecreates/footlight-app/assets/64489312/3f3257c6-cacf-45c1-b079-cbb4458bcb8d)

Second, I would like default values to be entered when no aspect ratio or width is added for both thumbnail and large (as shown above). Can you ask Dev or Suhail to put this in from the backend as part of the calendar preconfiguration.  See for example the NAC calendar (empty fields). 


![Image](https://github.com/culturecreates/footlight-app/assets/64489312/12dbf4dc-56e7-40db-b020-3825aa3ae458)

**Comment by troughc** [comment]
@SyamBabu-M there are two issues, one with Artbi where I can save without adding a large image aspect ratio

![Image](https://github.com/culturecreates/footlight-app/assets/64489312/3f3257c6-cacf-45c1-b079-cbb4458bcb8d)

Second, I would like default values to be entered when no aspect ratio or width is added for both thumbnail and large (as shown above). Can you ask Dev or Suhail to put this in from the backend as part of the calendar preconfiguration.  See for example the NAC calendar (empty fields). 

![Image](https://github.com/culturecreates/footlight-app/assets/64489312/12dbf4dc-56e7-40db-b020-3825aa3ae458)

Important side note: let's discuss the process of a superadmin setting up a default calendar. I will open a separate issue for this.

**Comment by dev-aravind** [comment]
@SyamBabu-M The backend changes are up in staging. Let me know if you have any questions.

cc @troughc

**Comment by troughc** [comment]
Tested: PASSED 
Two questions before deploying @SyamBabu-M  @dev-aravind 
In Artbi, the max width can be set to 0 and i can save this setting.
1. I think that would defeat the purpose (as the image would never show if the max width is 0 pixels), am I right? So what is happening when we set the max width to 0?
2. Should we disallow someone from setting a max width to 0 (or some other unreasonably small number)?

thanks - screenshot


![Image](https://github.com/culturecreates/footlight-app/assets/64489312/ed5ea52d-3576-41ee-aac0-7582a0665cbc)
